tests/test_pytest/test_plugin_schema/list.yaml

```diff
@@ -0,0 +1,26 @@
+interface: products
+family: list
+name: abi
+docstring: >
+  List of products for the ABI sensor
+
+  A list of products for the Advanced Baseline Imager on the GOES satellites.
+spec:
+  - interface: products
+    family: interpolator_algorithm_colormap
+    name: infrared
+    docstring: >
+      11um Infrared
+
+      11um infrared product with a multi-hue colorbar
+  - interface: products
```

Comment:
Do we have an example of a product list that uses "product_defaults" without including family/interface/docstring ?  With and without spec overrides?  I think that will be the most common use case.

Reply:
I just added one more test that does this in `tests/test_pytest/good/products/single.yaml`. I also removed the file referenced here. It was no longer used.